x86/src/Data/Macaw/X86/ArchTypes.hs

```diff
@@ -917,6 +952,12 @@ instance IsArchFn X86PrimFn where
       AESNI_AESDecLast x y -> sexprA "aesdeclast" [pp x, pp y]
       AESNI_AESKeyGenAssist x i -> sexprA "aeskeygenassist" [pp x, ppShow i]
       AESNI_AESIMC x -> sexprA "aesimc" [pp x]
+      SHA_sigma0 x -> sexprA "sigma0" [pp x]
+      SHA_sigma1 x -> sexprA "sigma1" [pp x]
+      SHA_Sigma0 x -> sexprA "Sigma0" [pp x]
+      SHA_Sigma1 x -> sexprA "Sigma1" [pp x]
+      SHA_Ch x y z -> sexprA "Ch" [pp x, pp y, pp z]
+      SHA_Maj x y z -> sexprA "Maj" [pp x, pp y, pp z]
```

Comment:
Could we prefix these names with `sha_` or something similar? If printed without context, they would look pretty mysterious